instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests/ui/routeline/RouteLineLayersTest.kt

```diff
@@ -372,6 +390,57 @@ class RouteLineLayersTest : BaseTest<BasicNavigationViewActivity>(
         countDownLatch.await()
     }
 
+    /**
+     * This test ensures that we're not crashing when parsing alternative routes metadata
+     * in a specific case (caught by crashlytics) that involved
+     * Nav Native reporting the [AlternativeRouteIntersection.geometryIndexInRoute] of
+     * [AlternativeRouteMetadata.forkIntersectionOfAlternativeRoute] to be out of bounds
+     * for the shape points collection of that particular route.
+     */
+    @Test
+    fun should_not_crash_when_fork_intersection_out_of_bounds_NAVAND_692() = sdkTest {
```

Comment:
Why do you need a ticket name in the test? It seems weird. You can retrace the steps of adding this test via git blame. What additional information does it give here? The name itself specifies what's being checked.

Reply:
Git history can easily be lost, or hard to trace back when files change locations or even worse repositories (we've been there). A regression test targets a specific occurrence that might require additional context, and since in this case the test itself is simple enough that it would pass without a problem if it was refactored in any way, I'd prefer keeping the context linked in code.